tests/e2e/stablehlo_ops/CMakeLists.txt

```diff
@@ -302,170 +302,6 @@ iree_check_single_backend_test_suite(
     "local"
 )
 
-iree_check_single_backend_test_suite(
-  NAME
-    check_cuda_graph
```

Comment:
FYI, my recent https://github.com/openxla/iree/pull/16086 is working as intended here :D

When 'cuda' was enabled and 'cuda2' was disabled: https://github.com/openxla/iree/actions/runs/7632069510/job/20791981080#step:7:1203
```
The following tests did not run:
	593 - iree/tests/e2e/stablehlo_ops/check_cuda2_graph_abs.mlir (Disabled)
	594 - iree/tests/e2e/stablehlo_ops/check_cuda2_graph_add.mlir (Disabled)
```

After the flip, when 'cuda2' was enabled' and 'cuda' was disabled:
https://github.com/openxla/iree/actions/runs/7632476397/job/20793221872#step:7:1221
```
The following tests did not run:
	471 - iree/tests/e2e/stablehlo_ops/check_cuda_graph_abs.mlir (Disabled)
	472 - iree/tests/e2e/stablehlo_ops/check_cuda_graph_add.mlir (Disabled)
```

Prior to that change, the disabled targets wouldn't appear in ctest output at all. Now it is pretty clear what is enabled or disabled. (I could have checked that on the flip PR itself, but I forgot :P)

Reply:
Ah, nice to know!